### PR TITLE
pythonPackages.leb128: init at 1.0.4

### LINKED ISSUE #130937: python-leb128
**Project description**

LEB128 or Little Endian Base 128 is a form of variable-length code compression used to store an arbitrarily large integer in a small number of bytes. LEB128 is used in the DWARF debug file format and the WebAssembly binary encoding for all integer literals.

This module makes it easy to encode/decode both signed and unsigned LEB128 integers.

**Metadata**

* homepage URL: https://pypi.org/project/leb128/
* source URL: https://github.com/mohanson/leb128
* license: MIT
* platforms: Linux, all Python

### PR DESCRIPTION
**Pull request by urlordjames**
###### Motivation for this change
resolves #130937

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/leb128/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  # fetchPypi doesn't include files required for tests
+  src = fetchFromGitHub {
+    owner = "mohanson";
+    repo = "leb128";
+    # repo lacks proper version tags
+    rev = "bffcb77";
+    sha256 = "1gbc7gag7g8g63aw05aj7q26r84cy9f5a9bs2f4qsflk2n3c1fly";
+  };
+
+  checkInputs = [ pytestCheckHook ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/leb128/default.nix:16]: Please add a pythonImportsCheck.

File: pkgs/development/python-modules/leb128/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+buildPythonPackage rec {
+  pname = "leb128";
+  version = "1.0.4";
+
+  # fetchPypi doesn't include files required for tests
+  src = fetchFromGitHub {
+    owner = "mohanson";
+    repo = "leb128";
+    # repo lacks proper version tags
+    rev = "bffcb77";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/leb128/default.nix:12]:
```suggestion
    rev = "bffcb77f71501c2972d1961c9559e47e7107d838";
```

Comment (urlordjames) [reply pkgs/development/python-modules/leb128/default.nix:12]: Thanks!

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 131012` at 31461fd4 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.leb128</li>
    <li>python39Packages.leb128</li>
  </ul>
</details>

**Pull request merged**